catalog/spatial/geocoding/spatial-geocoding-api/src/main/java/org/codice/ddf/spatial/geocoding/GeoCodingConstants.java

```diff
@@ -43,4 +46,15 @@
   public static final String DEFAULT_TAG = "gazetteer";
 
   public static final String COUNTRY_TAG = "country-shape";
+
+  public static final String GEONAMES_TAG = "geonames";
```

Comment:
❓ Do we need both the geonames and gazeteer tag?
I remember reading a discussion or comment related to this but don't remember the outcome.

Reply:
We are using the `gazetteer` tag to query for any gazetteer metacards, but we use the `geonames` tag to retrieve metacards that are specifically from GeoNames so we can perform CRUD operations on them (the same is true for the country-shape metacards).  In essence we need a way to distinguish them from each other, as well as a way to easily query for both.